### PR TITLE
BZ1999786: added a KCS ref link for system reserved values

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.7 (cherry pick to 4.6)
- [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1999786)
- [Preview](https://deploy-preview-40207--osdocs.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-resources-configuring.html#nodes-nodes-resources-configuring-setting_nodes-nodes-resources-configuring)

### CODE REVIEW

File: modules/nodes-nodes-resources-configuring-setting.adoc [hunk 1]
```diff
@@ -26,41 +26,7 @@ For example, to access the resources from `cluster.node22` node, you can enter:
...
-                    "usageBytes": 129671168,
-                    "workingSetBytes": 102416384
-                },
-                "name": "runtime"
-            }
-        ]
-    }
-}
-----
+For more details, refer to link:https://access.redhat.com/solutions/5843241[Recommended system-reserved values].
```
Comment (bobfuru) [inline modules/nodes-nodes-resources-configuring-setting.adoc:29]: Same suggestion as https://github.com/openshift/openshift-docs/pull/40206
```suggestion
For more details, refer to the link:https://access.redhat.com/solutions/5843241[recommended system-reserved values].
```

**Review by bobfuru — COMMENTED** [review]
One minor suggestions; otherwise, LGTM.

**Comment by bobfuru** [comment]
While it might be possible to cherrypick from 4.7 -> 4.6, we generally only cherrypick from the `main` branch. Would you mind opening a new related PR for the 4.6 version?

**Comment by Amrita42** [comment]
ok sure , I am curious to know we dont cherry pick from 4.7 to 4.6?

**Comment by bobfuru** [comment]
> ok sure , I am curious to know why we cant cherry pick from 4.7 to 4.6? is it an issue from the GitHub UI side ?

There's a note in our docs manual that we can only use auto cherrypick from `main`:
> NOTE: Use the /cherrypick command from only the main branch.

**Comment by vikram-redhat** [comment]
> NOTE: Use the /cherrypick command from only the main branch.

Sorry @bobfuru. My fault. That should be a suggestion only. I _recommend_ we only use it from the main branch but it is not a technical limitation. You can CP from any branch to any branch and it should be ok to do so in this case.

**Comment by bobfuru** [comment]
QE ack in related PR https://github.com/openshift/openshift-docs/pull/40206 so this LGTM

**Pull request merged**